### ISSUE TITLE
Be logged out on IDP side

### ISSUE THREAD

**Issue by llomgui** [nginxinc/kubernetes-ingress#4989]
**Is your feature request related to a problem? Please describe.**
With the current [oidc configuration](https://github.com/nginxinc/kubernetes-ingress/blob/main/internal/configs/oidc/openid_connect.js#L266), if you go on https://domain.org/logout, it only clears the cookies from Nginx perspective, not from IDP.
As describe in [Auth0 documentation](https://auth0.com/docs/authenticate/login/logout/log-users-out-of-auth0#how-it-works), you need to redirect the user on IDP's logout endpoint.

**Describe the solution you'd like**
The solution is to redirect the user on IDP's logout endpoint with the correct arguments.
On the IDP side, you can specify the Nginx logout endpoint /_logout and then Nginx will redirect to a final page https://domain.org/login.

Nginx teams already done that in a previous project. Not implemented on this one.
https://github.com/nginx-openid-connect/nginx-oidc-auth0/blob/main/oidc.js#L178
https://github.com/nginx-openid-connect/nginx-oidc-auth0/blob/main/oidc_nginx_server.conf#L132

```[tasklist]
### UACs
- [x] Test and validate changes in https://github.com/nginxinc/kubernetes-ingress/pull/4986 with [updates merged to NGINX OIDC repo](https://github.com/nginxinc/nginx-openid-connect/pull/96)
```

**Comment by brianehlert** [comment]
The OIDC reference implementation has been updated and NIC needs to be refreshed to align.
@danielnginx @shaun-nx

**Comment by j1m-ryan** [comment]
Hi @llomgui, this functionality will be available in `3.7.0`.
